spring-cloud-generator/generate-one.sh

```diff
@@ -106,9 +110,12 @@ add_module_to_pom () {
     echo "adding module $starter_artifactid to pom"
     sed -i "/$2/a\ \ \ \ <module>"$starter_artifactid"</module>" $1
     if [[ ${3:-1} -eq 1 ]]; then
+      # we need to remove any previously existing entries so the entry points to the
+      # right $monorepo_commitish's readme
+      sed -i "/$starter_artifactid/d" README.md
       # also write to spring-cloud-previews/README.md
       # format |name|distribution name|
-      echo -e "|$client_lib_name|com.google.cloud:$starter_artifactid|" >> README.md
+      echo -e "|$client_lib_name|com.google.cloud:$starter_artifactid|[README.md](https://github.com/googleapis/google-cloud-java/blob/$monorepo_commitish/$monorepo_folder/README.md)|" >> README.md
```

Comment:
@diegomarquezp feel free to move this discussion to a different PR when you separate out the changes, but what do you think of something like this? 
```suggestion
      echo -e "|[$monorepo_folder](https://github.com/googleapis/google-cloud-java/blob/$monorepo_commitish/$monorepo_folder/README.md)|com.google.cloud:$starter_artifactid|" >> README.md
```
| Client Library | Starter Maven Artifact |
|-|-|
| [java-language](https://github.com/googleapis/google-cloud-java/blob/v1.2.0/java-language/README.md)| com.google.cloud:google-cloud-language-spring-starter|

Reply:
Moving this to #1501.